packages/opentelemetry-semantic-conventions/src/experimental.ts

```diff
@@ -0,0 +1,6182 @@
+- [complete-multipart-upload](https://docs.aws.amazon.com/cli/latest/reference/s3api/complete-multipart-upload.html)
+- [list-parts](https://docs.aws.amazon.com/cli/latest/reference/s3api/list-parts.html)
+- [upload-part](https://docs.aws.amazon.com/cli/latest/reference/s3api/upload-part.html)
+- [upload-part-copy](https://docs.aws.amazon.com/cli/latest/reference/s3api/upload-part-copy.html).
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+*/
+export const ATTR_AWS_S3_UPLOAD_ID = 'aws.s3.upload_id';
+
+/**
+ * The GraphQL document being executed.
+ *
+ * Note: The value may be sanitized to exclude sensitive information.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_GRAPHQL_DOCUMENT = 'graphql.document';
+
+/**
+ * The name of the operation being executed.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_GRAPHQL_OPERATION_NAME = 'graphql.operation.name';
+
+/**
+ * The type of the operation being executed.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_GRAPHQL_OPERATION_TYPE = 'graphql.operation.type';
+
+/**
+ * Compressed size of the message in bytes.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_MESSAGE_COMPRESSED_SIZE = 'message.compressed_size';
+
+/**
+ * MUST be calculated as two different counters starting from `1` one for sent messages and one for received message.
+ *
+ * Note: This way we guarantee that the values will be consistent between different implementations.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_MESSAGE_ID = 'message.id';
+
+/**
+ * Whether this is a received or sent message.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_MESSAGE_TYPE = 'message.type';
+
+/**
+ * Uncompressed size of the message in bytes.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_MESSAGE_UNCOMPRESSED_SIZE = 'message.uncompressed_size';
+
+/**
+ * The stream associated with the log. See below for a list of well-known values.
+ *
+ * @experimental this attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const LOG_IOSTREAM_VALUES_STDOUT = 'stdout';
```

Comment:
The values didn't have a prefix before. Do we think they should have one now or is it ok to leave them out?

Reply:
I'm not sure exactly what you mean here.

Reply:
Like the attributes have the prefix `ATTR_` and metrics constants have the prefix `METRIC_`. The enum values don't have a prefix. They're just named by whatever the enum name was. Should it be `LOG_IOSTREAM_VALUES_STDOUT` or should it be `ENUM_LOG_IOSTREAM_VALUES_STDOUT`?

In @MSNev's constant version he had `LOGIOSTREAMVALUES_STDOUT`.